packages/abstraxion-core/tsup.config.ts

```diff
@@ -6,7 +6,7 @@ export default defineConfig((options: Options) => ({
   entry: ["src/index.ts"],
   format: ["esm", "cjs"],
   dts: true,
-  minify: false,
+  minify: true,
```

Comment:
Does this make it harder to debug?

Reply:
It can, yes, but this was a huge factor in dropping down those build sizes by nearly half. All ears on how we want to continue

Reply:
Interesting. This might be a follow on research project. I've always assumed the final bundler would be the one doing the minification.